packages/create-onchain/src/cli.ts

```diff
@@ -192,6 +475,85 @@ async function init() {
   console.log(' - npm run dev');
 }
 
+export function getArgs() {
+  const options = {
+    isHelp: false,
+    isVersion: false,
+    isGenerate: false,
+    isMiniKit: false,
+  };
+
+  // find any argument with -- or -
+  const arg = process.argv.find(
+    (arg) => arg.startsWith('--') || arg.startsWith('-'),
+  );
+  switch (arg) {
+    case '-h':
+    case '--help':
+      options.isHelp = true;
+      break;
+    case '-v':
+    case '--version':
+      options.isVersion = true;
+      break;
+    case '-g':
+    case '--generate':
+      options.isGenerate = true;
+      break;
+    case '-m':
+    case '--mini':
+      options.isMiniKit = true;
+      break;
+    default:
+      break;
+  }
+
+  return options;
+}
+
+async function init() {
+  const { isHelp, isVersion, isGenerate, isMiniKit } = getArgs();
+  if (isHelp) {
+    console.log(
+      `${pc.greenBright(`
+Usage:
+npm create-onchain [options]
+
+Creates an OnchainKit project based on nextJs.
+
+Options:
+--version, -v: Show version
+--mini, -m: Create a MiniKit project
```

Comment:
If we add more templates, these flags are gonna start clashing. I'd go with --template=blah personally, but don't feel super strongly about it